kubetest2/kubetest2-gce/deployer/up.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployer
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"k8s.io/klog"
+	"k8s.io/test-infra/kubetest2/pkg/exec"
+)
+
+func (d *deployer) Up() error {
+	klog.Info("GCE deployer starting Up()")
+
+	if err := d.verifyFlags(); err != nil {
+		return fmt.Errorf("up could not verify flags: %s", err)
+	}
+
+	if err := enableComputeAPI(d.GCPProject); err != nil {
+		return fmt.Errorf("up couldn't enable compute API: %s", err)
+	}
+
+	env := d.buildEnv()
+	script := filepath.Join(d.RepoRoot, "cluster", "kube-up.sh")
+	klog.Infof("About to run script at: %s", script)
+	klog.Infof("Running under environment: %+v", env)
```

Comment:
you might not want to do this. Be careful of dumping passwords and secrets. Optionally, you may want to use a verbosity flag + warning this could dump credentials

Reply:
Removed it. While the env being provided does not currently inherit from the OS, it could in future and it wouldn't be clear that this could dump secrets.